internal/cltest/factories.go

```diff
@@ -244,7 +247,7 @@ func NewRunLog(jobID string, addr common.Address, blk int, json string) ethtypes
 			services.RunLogTopic,
 			StringToHash("internalID"),
 			StringToHash(jobID),
-			common.BigToHash(big.NewInt(MinimumContractPayment)),
+			config.MinimumContractPayment.ToHash(),
```

Comment:
optional nitpick: I'm inclined to keep the const `MinimumContractPayment` because, unfortunately, `cltest.NewConfig()` spins up a whole new websocket server and is more costly. I understand that we don't want to refactor that in this PR.

Reply:
@dimroc I'm trying to avoid mixing types so would like to avoid the `MinimumContractPayment` `const`. How do you feel about adding something like `NewTestConfig` which is basically everything in `NewConfigWithWSServer` except `config.SetEthereumServer(wsserver)`?

Reply:
Hmm sure, I'm not opposed to that. I assume it wouldn't have a `cleanup` in the return statement.

I do just want to point out though that technically `MinimumContractPayment` doesn't have a type.

```
const MinimumContractPayment = 100
```

Notice there is no `:=` for type inference. In go, these const's are almost akin to a string replacement but with compile type checking. So types won't get mixed up.

>  if we made v a constant, even an untyped constant, we'd be back in forbidden territory:

```
    var u uint
    const v = -1
    u = uint(v) // Error: negative value
```

Not sure if that will sway your decision. You could just do `assets.NewLink(MinimumContractPayment)`. Further reading: https://blog.golang.org/constants

Reply:
@dimroc interesting.

How about the change I just pushed? It uses a `var` instead of a `const` so that can take our custom type.